hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/ContainerDataYaml.java

```diff
@@ -252,8 +252,9 @@ public Object construct(Node node) {
 
         //Needed this, as TAG.INT type is by default converted to Long.
         long layOutVersion = (long) nodes.get(OzoneConsts.LAYOUTVERSION);
-        ChunkLayOutVersion layoutVersion =
-            ChunkLayOutVersion.getChunkLayOutVersion((int) layOutVersion);
+        ContainerLayoutVersion layoutVersion =
+            ContainerLayoutVersion.getContainerLayoutVersion(
+                (int) layOutVersion);
```

Comment:
Nit: for consistency.

```suggestion
                (int) layoutVersion);
```